maven-scm-plugin/src/main/java/org/apache/maven/scm/plugin/TagMojo.java

```diff
@@ -139,12 +147,14 @@ public void execute()
             ScmRepository repository = getScmRepository();
             ScmProvider provider = getScmManager().getProviderByRepository( repository );
 
+            getLog().info( "Sign: '" + sign+ "'" );
```

Comment:
Please drop this, we don't log the rest anyway. The plugin framework logs everything if in debug mode.